site/src/pages/TemplateVersionEditorPage/TemplateVersionEditorPage.test.tsx

```diff
@@ -324,14 +324,13 @@ test("display pending badge and update it to running when status changes", async
 		},
 	};
 
-	let calls = 0;
+	let running = false;
 	server.use(
 		http.get(
 			"/api/v2/organizations/:org/templates/:template/versions/:version",
 			() => {
-				calls += 1;
 				return HttpResponse.json(
-					calls > 1 ? MockRunningTemplateVersion : MockPendingTemplateVersion,
+					running ? MockRunningTemplateVersion : MockPendingTemplateVersion,
```

Comment:
This test was failing because as we end up querying versions/{id} an extra time with this change. I think we should just manually update the status instead of relying on a specific number of api calls.

Reply:
Can you please leave a comment in the code about why we are using `running`? I can see my self not remembering this in two months.